labman/db/study.py

```diff
@@ -108,3 +108,108 @@ def num_samples(self):
                      WHERE study_id = %s"""
             TRN.add(sql, [self.id])
             return TRN.execute_fetchlast()
+
+    @property
+    def num_16S_amplified(self):
```

Comment:
The codebase is not actually restricted to 16S amplicons despite the nomenclature, can this be fixed so it's `number_amplicons_amplified`? Same with other instances of the use of 16S